website/src/client/utils/Appetize.ts

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useMemo, useState } from 'react';
+import type { SDKVersion } from 'snack-sdk';
+
+import constants from '../configs/constants';
+
+/**
+ * Get the default Appetize SDK config from the constants.
+ */
+export function getAppetizeConstants({
+  sdkVersion,
+  platform,
+  isEmbedded,
+}: {
+  sdkVersion: SDKVersion;
+  isEmbedded: boolean;
+  platform: 'android' | 'ios';
+}): AppetizeSdkConfig {
+  const values = isEmbedded
+    ? constants.appetize?.[sdkVersion]?.embed?.[platform]
+    : constants.appetize?.[sdkVersion]?.main?.[platform];
+
+  if (!values) {
+    throw new Error(
+      `No Appetize config found for SDK ${sdkVersion} and platform ${platform}, configure it in client/configs/constants.tsx`
+    );
+  }
+
+  return values;
+}
+
+const DEVICE_NAMES: Record<string, string> = {
+  iphone8: 'iPhone 8',
+  iphone8plus: 'iPhone 8 Plus',
+  iphone11pro: 'iPhone 11 Pro',
+  iphone12: 'iPhone 12',
+  iphone13pro: 'iPhone 13 Pro',
+  iphone13promax: 'iPhone 13 Pro Max',
+  iphone14pro: 'iPhone 14 Pro',
+  iphone14promax: 'iPhone 14 Pro Max',
+  iphone15pro: 'iPhone 15 Pro',
+  iphone15promax: 'iPhone 15 Pro Max',
+  // ipadair4thgeneration: 'iPad Air (4th gen)',
+  // ipadpro129inch5thgeneration: 'iPad Pro 12.9 (5th gen)',
+  // ipad9thgeneration: 'iPad (9th gen)',
+  nexus5: 'Nexus 5',
+  pixel4: 'Pixel 4',
+  pixel4xl: 'Pixel 4 XL',
+  pixel6: 'Pixel 6',
+  pixel6pro: 'Pixel 6 Pro',
+  pixel7: 'Pixel 7',
+  pixel7pro: 'Pixel 7 Pro',
+  // galaxytabs7: 'Galaxy Tab S7',
```

Comment:
These tablets are too big for the current panel size, this would also be an issue if we want to add a "rotate device" option. Maybe we should make the panel resizable somehow.

Reply:
See expo/snack#560